### PR TITLE
[5.5] Add more uncountable nouns

### PR DESCRIPTION
**Pull request by TheAlexLichter**

### CODE REVIEW

**Comment by jmarcher** [comment]
Sorry, but your list is wrong, there are quite a few words that are indeed countable.

File: src/Illuminate/Support/Pluralizer.php [hunk 1]
```diff
@@ -12,28 +12,45 @@ class Pluralizer
...
         'chassis',
+        'chess',
+        'commerce',
         'compensation',
+        'content',        
         'coreopsis',
+        'damage',
         'data',
         'deer',
+        'distribution',
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/Pluralizer.php]: This is definitely countable.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Illuminate/Support/Pluralizer.php [hunk 2]
```diff
@@ -12,28 +12,45 @@ class Pluralizer
      * @var array
      */
     public static $uncountable = [
+        'access',
+        'accommodation',
+        'advice',
```
Comment (jmarcher) [inline src/Illuminate/Support/Pluralizer.php]: This one is countable

File: src/Illuminate/Support/Pluralizer.php [hunk 3]
```diff
@@ -42,14 +59,20 @@ class Pluralizer
         'plankton',
         'pokemon',
         'police',
+        'quality',
```
Comment (jmarcher) [inline src/Illuminate/Support/Pluralizer.php]: This one is countable too

File: src/Illuminate/Support/Pluralizer.php [hunk 4]
```diff
@@ -42,14 +59,20 @@ class Pluralizer
...
+        'quantity',
```
Comment (jmarcher) [inline src/Illuminate/Support/Pluralizer.php]: This one too

**Comment by manniL** [comment]
I shouldn't make PR's late at night 😅   Thanks, @jmarcher and @GrahamCampbell. I removed those words.

**Comment by m1guelpf** [comment]
@manniL This is a breaking change and should go to the `master` branch. The pluralizer is used for getting the table Eloquent models use.

**Comment by manniL** [comment]
@m1guelpf Referring to other PRs #21324, #21306, #21236, this isn't considered as a breaking change.

**Comment by m1guelpf** [comment]
@manniL See discussion from https://github.com/laravel/framework/pull/21324#issuecomment-331607085

**Comment by GrahamCampbell** [comment]
Please send to master. There have been enough potentially breaking additions for people to deal with.

**Pull request closed without merging**